README.md

```diff
@@ -1,12 +1,15 @@
 # ExSecp256k1
 
-Rust Nif that wraps a couple functions from the [libsecp256k1](https://github.com/paritytech/libsecp256k1) rust library. It only wraps secp256k1 functions used in Ethereum.
+Rust Nif that wraps a couple functions from the
+[libsecp256k1](https://github.com/paritytech/libsecp256k1) Rust library. It
+only wraps `secp256k1` function used in Ethereum.
 
 ## Installation
 
 `ex_secp256k1` requires Rust to be installed.
 
-The package can be installed by adding `ex_secp256k1` to your list of dependencies in `mix.exs`:
+The package can be installed by adding `ex_secp256k1` to your list of
```

Comment:
I don't think it has to be split into multiple lines. It looks ok https://hexdocs.pm/ex_secp256k1/readme.html. the same comment for line 3, 22, 27, 52

Reply:
> I don't think it has to be split into multiple lines. It looks ok https://hexdocs.pm/ex_secp256k1/readme.html. the same comment for line 3, 22, 27, 52

@ayrat555 Done.